README.md

```diff
@@ -1,23 +1,71 @@
 
 [![Build Status for the Core backend](https://github.com/devitocodes/devito/workflows/CI-core/badge.svg)](https://github.com/devitocodes/devito/actions?query=workflow%3ACI-core)
 [![Build Status with MPI](https://github.com/devitocodes/devito/workflows/CI-mpi/badge.svg)](https://github.com/devitocodes/devito/actions?query=workflow%3ACI-mpi)
 [![Code Coverage](https://codecov.io/gh/devitocodes/devito/branch/master/graph/badge.svg)](https://codecov.io/gh/devitocodes/devito)
+[![Slack Status](https://img.shields.io/badge/chat-on%20slack-%2336C5F0)](https://opesci-slackin.now.sh)
+
+[Devito](http://www.devitoproject.org) is a Python package to implement
+optimized stencil computation from high-level symbolic problem definitions.
+Devito builds on [SymPy](http://www.sympy.org/en/index.html) and employs
+automated code generation and just-in-time (JIT) compilation to execute
+optimized computational kernels on several computer platforms, including
+CPUs, GPUs, and clusters thereof.
+
+- [About Devito](#about-devito)
+- [Installation](#installation)
+- [Resources](#resources)
+- [Performance](#performance)
+- [Get in touch](#get-in-touch)
+
+## About Devito
+
+Devito provides a functional language to implement sophisticated operators that
+can be made up of multiple stencil computations, boundary conditions, sparse
+operations (e.g., interpolation), and much more.  A typical use case is
+explicit finite difference methods for approximating partial differential
+equations. For example, a 2D diffusion operator may be implemented with Devito
+as follows
+
+```python
+>>> grid = Grid(shape=(10, 10))
+>>> f = TimeFunction(name='f', grid=grid, space_order=2)
+>>> eqn = Eq(f.dt, 0.5 * f.laplace)
+>>> op = Operator(Eq(f.forward, solve(eqn, f.forward)))
+```
 
-[Devito](http://www.devitoproject.org) is a software to
-implement optimised finite difference (FD) computation from
-high-level symbolic problem definitions. Starting from symbolic
-equations defined in [SymPy](http://www.sympy.org/en/index.html),
-Devito employs automated code generation and just-in-time (JIT)
-compilation to execute FD kernels on multiple computer platforms.
-
-## Get in touch
+An `Operator` generates low-level code from an ordered collection of `Eq` (just
+one in the example above). This code may also be compiled and executed
 
-If you're using Devito, we would like to hear from you. Whether you
-are facing issues or just trying it out, join the
-[conversation](https://opesci-slackin.now.sh).
+```python
+>>> op(t=timesteps)
+```
 
-## Quickstart
+There is virtually no limit to the complexity of an `Operator` -- the Devito
+compiler will automatically analyze the input, detect and apply optimizations
+(including single- and multi-node parallelism), and eventually generate code
+with suitable loops and expressions.
+
+Key features include:
+
+* A functional language to express finite difference operators.
+* Straightforward mechanisms to dynamically adjust the discretization.
```

Comment:
What exactly are you referring to here by 'dynamically' adjust the discretization?

Reply:
fair point, I think "dynamically" can be dropped as it doesn't add much. In fact, it may just create confusion.

Reply:
> In fact, it may just create confusion.

Indeed. I think dropping it is fine.

Reply:
done